timerangepicker/src/main/java/nl/joery/timerangepicker/TimeRangePicker.kt

```diff
@@ -277,28 +285,62 @@ class TimeRangePicker @JvmOverloads constructor(
         postInvalidate()
     }
 
+    private var gradientPositionsCache = FloatArray(2)
+    private var gradientColorsCache = IntArray(2)
+    private val gradientMatrixCache = Matrix()
```

Comment:
Move them to the top, where their friends are 😃. Also, they should probably start with an underscore, since that's how all other private fields are named.

Not sure what the Kotlin way is of naming private fields.